tests/data/test_loader.py

```diff
@@ -1,16 +1,15 @@
 from ert_data import loader
```

Comment:
From what I can see we have the following functions:

```
test_load_general_data
test_load_general_obs
test_load_block_data
test_load_summary_data
test_load_summary_obs
```

Should we complete this by adding:

```
test_load_block_obs
test_load_general_response
test_load_summary_response
test_load_block_response
```

Reply:
I will review the whole testing scheme 👍